workflow-templates/run-nextstrain-build-on-aws-batch.yaml

```
@@ -0,0 +1,28 @@
+name: Run Nextstrain build on AWS Batch
+
+on:
+  workflow_dispatch:
+    inputs:
+      image:
+        description: "Specific starter image to use for the build. Will override the default image for `nextstrain build`."
+        required: false
+        type: string
+
+jobs:
+  deploy-build-to-aws-batch:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: nextstrain/.github/actions/run-nextstrain-build-on-aws-batch@master
+        with:
+          runtime-envvar-names: "GITHUB_RUN_ID PAT_GITHUB_DISPATCH SLACK_CHANNELS SLACK_TOKEN"
```

_non-blocking_

One nicer way to format this in YAML would be:

```suggestion
          runtime-envvar-names: >-
            GITHUB_RUN_ID
            PAT_GITHUB_DISPATCH
            SLACK_CHANNELS
            SLACK_TOKEN
```